plugins.d/python.d.plugin

```diff
@@ -479,18 +504,15 @@ def parse_cmdline(directory, *commands):
             DEBUG_FLAG = True
             mods.append(cmd.replace(".chart.py", ""))
         else:
-            DEBUG_FLAG = False
             try:
-                interval = int(cmd)
+                update_every = int(cmd)
             except ValueError:
-                pass
+                update_every = UPDATE_EVERY
 
     debug("started from", commands[0], "with options:", *commands[1:])
-    if len(mods) == 0 and DEBUG_FLAG is False:
-        interval = None
 
-    return {'interval': interval,
-            'modules': mods}
+    UPDATE_EVERY = update_every
+    return {'modules': mods}
```

Comment:
Now it is not needed to wrap `mods` into dict. Just change lines in `run()` from:
```
out = parse_cmdline(MODULES_DIR, *sys.argv)
modules = out['modules']
```
to:
```
modules = parse_cmdline(MODULES_DIR, *sys.argv)
```